### DISCUSSION TITLE
dllcall

### DISCUSSION THREAD

**Discussion by Praisemeow** [spyoungtech/ahk#158] [Q&A] [answered]
is there no dllcall yet? if there no then would it be at all?
ty for answer

**Comment by spyoungtech** [comment, marked as answer]
Right now, DllCall is not implemented, partly because Python provides this functionality in the ctypes module. Similarly Run is not implemented because Python provides the same functionality in the `subprocess` module.

There's currently no plan to add dllcall, but if you and others think it would be useful to have, I may consider adding it.
